mm-go-irckit/userbridge.go

```diff
@@ -885,3 +930,96 @@ func (u *User) updateLastViewed(channelID string) {
 		u.br.UpdateLastViewed(channelID)
 	}()
 }
+
+func (u *User) saveLastViewed(statePath string) {
+	// We only want to save or dump out saved lastViewedAt on new
+	// messages after X time (default 5mins).
+	saveInterval := int64(300000)
+	val, err := time.ParseDuration(u.v.GetString(u.br.Protocol() + ".lastviewedsaveinterval"))
+	if err == nil {
+		saveInterval = val.Milliseconds()
+	}
```

Comment:
This can be refactored the same as with the `loadLastViewedState`, make the saveInterval a constant and do like below, this makes it imo more readable because of the `err != nil` flow, but it's nitpicking, so your call for this.

```
var saveInterval int64
val, err := time.ParseDuration(u.v.GetString(u.br.Protocol() + ".lastviewedsaveinterval"))
if err != nil {
    saveInterval = defaultSaveInterval
} else {
    saveInterval = val.Milliseconds()
}
```

Reply:
Fixed